CMakeLists.txt

```
@@ -57,14 +57,20 @@ add_library(beman::any_view ALIAS beman.any_view)
 
 set_target_properties(
     beman.any_view
-    PROPERTIES VERIFY_INTERFACE_HEADER_SETS ON EXPORT_NAME any_view
+    PROPERTIES
+        # cmake-format: off
+        VERIFY_INTERFACE_HEADER_SETS ON
+        EXPORT_NAME any_view
 )
 
 target_sources(
     beman.any_view
     PUBLIC
         FILE_SET HEADERS
-        BASE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/include include
+        BASE_DIRS
+            # cmake-format: off
+            ${CMAKE_CURRENT_BINARY_DIR}/include
```

Question: if `${CMAKE_CURRENT_BINARY_DIR}` were to have spaces, would this still be correct or would it need to be wrapped in quotes like this?
```suggestion
            "${CMAKE_CURRENT_BINARY_DIR}/include"
```

---

It would still be correct, unquoted argument expansion does not split the string it is happening within.

Ie, the evaluation works like this: evaluate CMakeLang -> Expand Variables

This does not happen: evaluate CMakeLang -> Expand Variables -> Re-evaluate CMakeLang with expanded variables.

For strings that otherwise contain no whitespace or escape characters, to quote or not is a style choice. Beman has no guidance on it, and its your library, so do whatever makes "this is a string" obvious to you.